plugins/feeds/public/feodo_tracker.py

```diff
@@ -89,35 +89,36 @@ def analyze(self, item):
 
                 soup = BeautifulSoup(s, 'html.parser')
                 res = soup.find_all('table')
```

Comment:
Maybe use `find_all('table.sortable')` to just focus on the table that contains the interesting information?